stats.tex

```diff
@@ -0,0 +1,50 @@
+%% Stats
+The \verb|sympy.stats| module provides random variable types and methods for
+determination of statistical properties of expressions involving random
```

Comment:
Being a technical paper, I suggest to mention some internal stuff. Most of the algorithms in _sympy.stats_ are about generating integrals/summations. They can be shown with `evaluate=False`.

The term _properties_ is too general, it could mean anything. What about _symbolic values_ instead?